### PR TITLE
Upgrade Gem code

### PR DESCRIPTION
**Pull request by vcastellm**
Update gem generated code

It now uses Addressable::URI

### CODE REVIEW

**Review by LordSarevok — COMMENTED** [review]
I assume that this code comes from another place. Right? Or your wrote that in one hour?

**Comment by victorcoder** [comment]
@LordSarevok 🤫  swagger codegen is my secret to pretend I'm a 10x developer 😂

**Review by LordSarevok — APPROVED** [review]

**Review by dagoonline — APPROVED** [review]
LGTM! :rocket: :trollface:

File: lib/dkron-rb/api_client.rb [hunk 1]
```diff
@@ -263,7 +265,7 @@ def sanitize_filename(filename)
     def build_request_url(path)
       # Add leading and trailing slashes to path
       path = "/#{path}".gsub(/\/+/, '/')
-      URI.encode(@config.base_url + path)
+      Addressable::URI.encode(@config.base_url + path)
```
Comment (beetlegius) [inline lib/dkron-rb/api_client.rb:268]: :clap:

**Review by beetlegius-jt — APPROVED** [review]

**Pull request merged**